### ISSUE TITLE
displaying the amount in white like minecraft ?

### ISSUE THREAD

**Issue by rom1504** [deathcap/inventory-window#28]
example at http://minecraft.gamepedia.com/Template:Crafting

(not very important, but it looks a bit better)

**Comment by deathcap** [comment]
Good idea I've been wanting to do this, but the text is not just white, rather it has a black shadow as well so it shows up better on lighter backgrounds. Should be possible with some fancy CSS text properties (shadow? emboss?).
